### DISCUSSION TITLE
Saving project is different from what the project print

### DISCUSSION THREAD

**Discussion by pengzhulin** [PMEAL/OpenPNM#2886] [Q&A]
Hi Professor Jeff Gostick. 
I am trying to save my project. However, the save result (vtk) is not same to the printed one. Do you know about

Here we use the basic example. https://openpnm.org/examples/tutorials/09_simulating_invasion.html. How can we save the InvasionPercolation (ip) results to vtk file which can be used in paraview or other visulization software? 

I use op.io._vtk.project_to_vtk(ip.project, filename="test") , but it fails to save the results.

I use "print(ip)"
ip_01 : <openpnm.algorithms.InvasionPercolation at 0x25cfcae1950>
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
  #  Properties      ----                                             Valid Values
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
  2  pore.invasion_sequence                                        3125 / 3125
  3  throat.invasion_sequence                                      8500 / 8500
  4  throat.entry_pressure                                         8500 / 8500
  5  throat.sorted                                                 8500 / 8500
  6  throat.order                                                  8500 / 8500
  7  throat.invasion_pressure                                      8500 / 8500
  8  pore.invasion_pressure                                        3125 / 3125
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
  #  Labels         --                      Assigned Locations
――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――
  2  pore.all                                                             3125
  3  throat.all                                                           8500
  4  pore.bc.inlet                                                           1
  5  pore.bc.outlet                                                        125
  6  pore.trapped                                                         1153
  7  throat.trapped                                                       6529

and I save vtk result by using 
op.io._vtk.project_to_vtk(ip.project, filename="test") 

![image](https://github.com/PMEAL/OpenPNM/assets/62170175/f05637b0-36b1-49ae-bd0d-3fb6177f5ff3)

Thank you very much!

Best regards,
Pengzhu

**Comment by jgostick** [comment]
We decided not to include the results of algorithms in the VTK output since this would really clutter up the results, so if you want to see the algorithm's output in paraview you need to do something like `phase.update(alg)` which will copy all the arrays on `alg` onto `phase`.  Hope that helps.
